cockroachcloud/client-certs-dedicated.md

```diff
@@ -0,0 +1,403 @@
+    ...\n-----END CERTIFICATE-----",
+    }
+    ~~~
+</section>
+
+<section class="filter-content" markdown="1" data-scope="tf">
+
+Update the `cockroach_client_ca_cert` block in your Terraform template, then run `terraform apply`.:
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+resource "cockroach_client_ca_cert" "yourclustername" {
+  id = cockroach_cluster.example.id
+  x509_pem_cert = file("cockroach_client_ca_cert.json")
+}
+~~~
+</section>
+
+## Delete the certificate authority (CA) certificate for a dedicated cluster
+
+This section shows how to remove the configured CA certificate from the cluster. Afterward, clients can no longer authenticate with certificates signed by this CA certificate.
+
+{{site.data.alerts.callout_success}}
+Managing the certificate authority (CA) certificate for a {{ site.data.products.dedicated }} cluster requires the [Cluster Administrator](authorization.html#cluster-administrator) or [Org Administrator (legacy)](authorization.html#org-administrator-legacy) Organization role.
+{{site.data.alerts.end}}
+
+<div class="filters clearfix">
+  <button class="filter-button page-level" data-scope="api">Using the API</button>
+  <button class="filter-button page-level" data-scope="tf">Using Terraform</button>
+</div>
+<p></p>
+
+<section class="filter-content" markdown="1" data-scope="api">
+
+1. Submit the asynchronous `DELETE` request, supplying your cluster ID, API key, and the path to the certificate JSON with your CA certificate, as described in [Create the certificate authority (CA) certificate](#create-the-certificate-authority-ca-certificate).
+
+    A `200` successful response code indicates that the asynchronous request was successfully submitted, but does not guarantee that the operation (configuring the CA certificate) successfully completed. You must confirm success with a follow-up `GET` request, as described in the next step.
+ 
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl --request DELETE \
+      --url ${COCKROACH_SERVER}/api/v1/clusters/${CLUSTER_ID}/client-ca-cert \
+      --header "Authorization: Bearer ${API_KEY}"
+    ~~~
+
+    ~~~txt
+    200 OK
+    ~~~
+
+1. Confirm success of the operation with the following `GET` request.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl --request GET \
+      --url ${COCKROACH_SERVER}/api/v1/clusters/${CLUSTER_ID}/client-ca-cert \
+      --header "Authorization: Bearer ${API_KEY}"
+    ~~~
+
+    `PENDING` indicates that the operation is still in process.
+    
+    ~~~txt
+    {
+      "status": "PENDING",
+      "x509_pem_cert": ""
+    }
+    ~~~
+    
+    `IS_SET` indicates that the operation completed successfully, confirming the configured public CA cert.
```

Comment:
This is the DELETE operation, so shouldn't be IS_SET, right? NOT_SET would be the correct response. Check this section - https://docs.google.com/document/d/1uyQuPmJ4NawuvaGTODr8AVArh7P-5TrdnWxELD6ppWI/edit#heading=h.47j5fsmw3vlu

Reply:
Thanks